src/workspaces/list/NoContentMessage.test.tsx

```diff
@@ -48,7 +32,19 @@ describe('NoContentMessage', () => {
 
   it('displays no workspaces message if there are no workspaces and filters tab is myWorkspaces', () => {
     // Arrange
-    const props = { loadingWorkspaces: false, workspaces: noWorkspaces, filters: keywordFilters };
+    const workspaces: CategorizedWorkspaces = {
+      myWorkspaces: [],
+      newAndInteresting: [],
+      featured: [],
+      public: [],
+    };
+    const filters: WorkspaceFilterValues = {
+      keywordFilter: 'nothingMatchesThis',
```

Comment:
Do you mind removing the keywordFilter to make it clear that it's not related to this test case?